CMakeLists.txt

```diff
@@ -526,6 +526,7 @@ set(scylla_sources
     repair/writer.cc
     repair/row.cc
     repair/row_level.cc
+    repair/row_level_repair.cc
```

Comment:
It's strange and confusing to have repair/row_level.cc and repair/row_level_repair.cc. Nobody will know what's supposed to go into which file.

Reply:
I suggest to rename row_level.cc to rpc_handler.cc since it mainly handles rpc related code. We can keep row_level_repair.cc as it is.

Reply:
I also was looking at it and thinking if I shall rename it or left intact to provide better backward history compatibility. I will rename it then.